notebook2/notebook/notebook.py

```diff
@@ -75,11 +72,13 @@ def read_string(f):
 
 POD_PORT = 8888
 
+SECRET_KEY = read_string('/notebook-secrets/secret-key')
+SECURE_COOKIE_FLAG = os.environ.get("NOTEBOOK_DEBUG") != "1"
```

Comment:
prefer `USE_SECURE_COOKIE` as it unambiguously conveys that `True` means the cookie is secure.